### PR TITLE
Ebnt 301 standalone build

### PR DESCRIPTION
**Pull request by mike0sv**

### CODE REVIEW

File: src/ebonite/ext/docker/builder.py [hunk 1]
```diff
@@ -11,7 +10,7 @@ class DockerBuilder(BuilderBase):
     """Builder implementation to build docker images"""
     @validate_kwargs
     def create_image(self, name: str, environment: DockerEnv, tag: str = 'latest', repository: str = None,
-                     **kwargs) -> Image.Params:
+                     **kwargs) -> DockerImage:
```
Comment (ElmiRakh) [inline src/ebonite/ext/docker/builder.py:13]: Wouldn't it cause IDE warnings about typing mismatch?

File: src/ebonite/ext/docker/helpers.py [hunk 2]
```diff
@@ -1,71 +1,55 @@
...
+    :param tag: image tag
+    :param repository: image repository
+    :param force_overwrite: wheter to force overwrite existing image
+    :parma kwargs: additional arguments for DockerBuilder.build_image
     """
-    with create_docker_client(check=False) as c:
-        return _is_docker_running(c)
-
-
-_docker_host_lock = Lock()
```
Comment (ElmiRakh) [inline src/ebonite/ext/docker/helpers.py:35]: Where did Lock() go? Or it is reduntant?

File: src/ebonite/ext/docker/helpers.py [hunk 3]
```diff
@@ -1,71 +1,55 @@
...
-@contextmanager
-def create_docker_client(docker_host: str = '', check=True) -> docker.DockerClient:
-    """
-    Context manager for DockerClient creation
-
-    :param docker_host: DOCKER_HOST arg for DockerClient
-    :param check: check if docker is available
-    :return: DockerClient instance
+    env = env or DockerEnv()
+    source = BuildableAnalyzer.analyze(obj, sever=server)
```
Comment (ElmiRakh) [inline src/ebonite/ext/docker/helpers.py:25]: typo in server?

File: src/ebonite/ext/docker/builder.py (hunk 1 above)
Comment (mike0sv) [reply src/ebonite/ext/docker/builder.py:13]: no

File: src/ebonite/ext/docker/helpers.py (hunk 3 above)
Comment (mike0sv) [reply src/ebonite/ext/docker/helpers.py:25]: lol

**Pull request merged**